tests/storage/cdi_import/test_import_http.py

```diff
@@ -532,29 +533,33 @@ def test_blank_disk_import_validate_status(data_volume_multi_storage_scope_funct
 
 
 @pytest.mark.parametrize(
-    "dv_from_http_import",
+    "data_volume_multi_storage_scope_function",
     [
         pytest.param(
             {
                 "dv_name": "cnv-3065",
-                "file_name": Images.Cdi.QCOW2_IMG,
-                "source": HTTPS,
-                "size": "100Mi",
-                "configmap_name": INTERNAL_HTTP_CONFIGMAP_NAME,
+                "source": HTTP,
+                "image": f"{Images.Alpine.DIR}/{Images.Alpine.QCOW2_IMG}",
```

Comment:
why we need the alpine image?

Reply:
I don't want to hold this PR, but wanted to hear from storage team - may be we can replace cirros with fedora?
@jpeimer